caqti-miou-unix.opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+name: "caqti-miou"
+maintainer: "Petter A. Urkedal <paurkedal@gmail.com>"
+authors: "Petter A. Urkedal <paurkedal@gmail.com>"
+license: "LGPL-3.0-or-later WITH LGPL-3.0-linking-exception"
+homepage: "https://github.com/paurkedal/ocaml-caqti/"
+doc: "https://paurkedal.github.io/ocaml-caqti/index.html"
+bug-reports: "https://github.com/paurkedal/ocaml-caqti/issues"
+depends: [
+  "caqti" {>= "2.1.0" & < "2.2.0~"}
+  "dune" {>= "3.9"}
+  "miou" {>= "0.3.0"}
+  "logs"
+  "ocaml" {>= "5.0.0~"}
+  "alcotest" {with-test & >= "1.5.0"}
+  "caqti-driver-sqlite3" {with-test}
+  "cmdliner" {with-test & >= "1.1.0"}
+  "mirage-crypto-rng-miou-unix" {with-test}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
+]
+dev-repo: "git+https://github.com/paurkedal/ocaml-caqti.git"
+synopsis: "Miou support for Caqti"
```

Comment:
To elaborate, I have bundled the unix packages with the main packages until now, I think originally dune to the fact that lwt does so.  I don't want to change that practise at the moment, since I'm trying to limit the number of packages, and I think it's a good fit for miou, anyway, since it also includes a unix sub-library.